LayoutTests/http/tests/resourceLoadStatistics/website-data-removal-for-site-navigated-to-with-link-decoration.html

```diff
@@ -5,17 +5,17 @@
     <script src="resources/util.js"></script>
 </head>
 <body onload="setTimeout('runTest()', 0)">
-<div id="description">Check that non-cookie website data gets removed after a navigation with link decoration from a prevalent resource.</div>
+<div id="description">Check that script-writeable website data gets removed after a navigation with link decoration from a prevalent resource.</div>
 <br>
 <div id="output"></div>
 <br>
 <script>
     testRunner.waitUntilDone();
     testRunner.dumpAsText();
 
-    const httpOnlyCookieName = "http-only-cookie";
-    const serverSideCookieName = "server-side-cookie";
-    const clientSideCookieName = "client-side-cookie";
+    const httpOnlyCookieName = "http_only_cookie";
+    const serverSideCookieName = "server_side_cookie";
+    const clientSideCookieName = "client_side_cookie";
```

Comment:
Is this some whitespace or linefeed change?

Reply:
The change here is replacing "-" with "_" in the string, but as I was writing an explanation about why this was needed I realized it's not needed.